server/Mistakes.Journal.Api/Api/Shared/MistakeHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Linq;
+using Mistakes.Journal.Api.Api.Shared.RequestsParameters;
+using Mistakes.Journal.Api.Logic.Mistakes.Models;
+
+namespace Mistakes.Journal.Api.Api.Shared
+{
+    public static class MistakeHelper
```

Comment:
I would name it MistakesExtensions

Reply:
You mean instead of MJException? It sounds like exception which that relates to Mistake(s), if not MJException when it's better to name it WeatherException